recipes/libmodulemd/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "libmodulemd" %}
+{% set version = "2.9.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/fedora-modularity/libmodulemd/releases/download/libmodulemd-{{ version }}/modulemd-{{ version }}.tar.xz
+  sha256: cb86b1dad4f1578895225ba4ee435dbb7d75262898f69a08507b01759bfc81ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - pkg-config
+    - ninja
+  host:
+    - file
+    - glib
+    - yaml
+    - rpm-tools
+    - gobject-introspection
+    - pygobject
+    - meson
+    - help2man
+  run:
+    - python
```

Comment:
Does the way you are using Python here actually work? I mean, this is compiled C but the Python bits are independent of that. Will conda understand how to install this in a different version of Python?
I was under the impression that one should always put python as a host & run dependency…but maybe that's the point here of putting it only into run?